api/src/main/java/org/apache/cloudstack/api/command/admin/vm/ImportUnmanagedInstanceCmd.java

```diff
@@ -84,9 +84,16 @@ public class ImportUnmanagedInstanceCmd extends BaseAsyncCmd {
     @Parameter(name = ApiConstants.NAME,
             type = CommandType.STRING,
             required = true,
-            description = "the hypervisor name of the instance")
+            description = "the name of the instance as it is known to the hypervisor")
     private String name;
 
+    @Parameter(name = ApiConstants.SERVICE_OFFERING_ID,
+            type = CommandType.UUID,
+            entityType = ServiceOfferingResponse.class,
+            required = true,
+            description = "the ID of the service offering for the virtual machine")
+    private Long serviceOfferingId;
+
```

Comment:
Is this refactoring needed?

Reply:
I thought it was a harmless refactoring as it moves all required params together. I can change back if you think it doesn't add value?